dashboard/app/assets/javascripts/soundEffects.js

```diff
@@ -158,6 +165,13 @@ Sound.prototype.play = function (options) {
 
   if (this.reusableBuffer) {
     this.playableBuffer = this.newPlayableBufferSource(this.reusableBuffer, options);
+
+    // Hook up on-ended callback, although browser support may be limited.
+    // Don't make anything depend on this callback happening.
+    if (options.onEnded) {
+      this.playableBuffer.onended = options.onEnded;
+    }
```

Comment:
Callbacks are easy in WebAudio, though I guess if you want more than one you're out of luck.